setup.py

```diff
@@ -51,15 +51,19 @@ def read(fname):
     long_description=read('README.md'),
```

Comment:
Please also add:

```
long_description_content_type="text/markdown",
```

PyPI should then render our `README.md` correctly.

Reply:
Good idea! Done! Travis will be busy for a while now.